### DISCUSSION TITLE
Not all my managers are being picked up

### DISCUSSION THREAD

**Discussion by gregoryduckworth** [renovatebot/renovate#9289] [Request Help] [answered]
I have a created a basic repository https://github.com/gregoryduckworth/renovate-multiple which shows the issue I'm having where the `package.json` file is not being picked up. The other various managers pick up the `docker-compose`, `dockerfile` and `build.gradle` file without any issues.

All I have done is extend the base config with a few tweaks, have I missed something? 

The logs show that it isn't being picked up as a manager:
```
DEBUG: Using file match: ^.travis.yml$ for manager travis (repository=gregoryduckworth/renovate-multiple)
DEBUG: Matched 3 file(s) for manager docker-compose: docker-compose.component.yml, docker-compose.override.yml, docker-compose.yml (repository=gregoryduckworth/renovate-multiple)
DEBUG: Matched 1 file(s) for manager dockerfile: Dockerfile (repository=gregoryduckworth/renovate-multiple)
DEBUG: Matched 1 file(s) for manager gradle: build.gradle (repository=gregoryduckworth/renovate-multiple)
DEBUG: gradle-lite is disabled (repository=gregoryduckworth/renovate-multiple)
```

**Comment by rarkins** [comment, marked as answer]
Scroll up a little in that log and you should see it printing the ignored repo patterns. When you extend `config:base`, it includes this preset: https://docs.renovatebot.com/presets-default/#ignoremodulesandtests

There's multiple ways around this, but adding `"ignorePresets": [":ignoreModulesAndTests"]` is probably easiest.

**Reply by gregoryduckworth** [reply]
You hero! It had been bothering me for ages!
